tox.ini

```diff
@@ -33,6 +33,7 @@ changedir=
 commands=
   vagrant up --no-provision {posargs:--provider=virtualbox}
   bash {toxinidir}/tests/scripts/generate_ssh_config.sh {changedir}
+  cp {toxinidir}/sites/site.yml.sample {toxinidir}/site.yml.sample
```

Comment:
I don't think this line is needed, it could just be changed in the ansible-playbook line below.

```
ansible-playbook -vv -i {changedir}/hosts {toxinidir}/sites/site.yml.sample
```

I'm also changing how this works in https://github.com/ceph/ceph-ansible/pull/1138 to use an environment variable.